scripts/anvil_tools/README.md

```diff
@@ -0,0 +1,101 @@
+# AnVIL Tools/Scripts
+
+------------------------
+#### This directory contains scripts/tools to automate steps in the AnVIL data delivery workspace creation process.
+------------------------
+
+### Setup
+    Follow the steps below to set up permissions prior to running any of the listed scripts:
+    1. `gcloud auth login username@broadinstitute.org`
+    2. `gcloud auth application-default login`
+        Select your broadinstitute.org email address when window opens.
+
+### Scripts
+
+#### **add_user_to_workspace.py**
+##### Description
+    Update/add user/group to workspace ACL (as READER, WRITER, OWNER). 
+    
+    Input is a .tsv file with 3 columns:
+        1. "workspace_name"
+        2. "email"
+        3. "accessLevel" - (READER, WRITER, or OWNER)
+    Output:
+        NA
```

Comment:
is there a success code?

Reply:
removed output blocks when script writes success/fail to stdout. these scripts will eventually be modified to write an output file and will add back in output sections then.

Reply:
👍